src/System.Private.CoreLib/shared/System/Globalization/DateTimeFormatInfo.cs

```diff
@@ -1707,50 +1645,40 @@ private static string[] GetMergedPatterns(string[] patterns, string defaultPatte
         internal const string RoundtripFormat = "yyyy'-'MM'-'dd'T'HH':'mm':'ss.fffffffK";
         internal const string RoundtripDateTimeUnfixed = "yyyy'-'MM'-'ddTHH':'mm':'ss zzz";
 
-        // Default string isn't necessarily in our string array, so get the
-        // merged patterns of both
+        /// <summary>
+        /// Default string isn't necessarily in our string array, so get the
+        /// merged patterns of both
+        /// </summary>
         private string[] AllYearMonthPatterns
         {
-            get
-            {
-                return GetMergedPatterns(this.UnclonedYearMonthPatterns, this.YearMonthPattern);
-            }
+            get => GetMergedPatterns(this.UnclonedYearMonthPatterns, this.YearMonthPattern);
```

Comment:
>get [](start = 12, length = 3)

do we need get at all here? this apply to similar changes like that

Reply:
Fixed